### DISCUSSION TITLE
K2 + AO centrifuge missing

### DISCUSSION THREAD

**Discussion by josiahwilkinson** [Xman1109/Atomic_Overhaul#9] [General]
Hello,

I am sorry, but I am having some trouble with the mod. I am playing with K2 + AO + Brevven's materials (not sure if they impact this issue). I am unable to create a centrifuge (by hand or assembler), and the tech tree does not show it anywhere as being unlockable. The Uranium Processing tech shows "Base mod > Atomic Overhaul > Krastorio 2" with just the regular uranium processing and fuel cell, but no centrifuge. The Nuclear Power tech (if it's supposed to be there) shows "Base mod > Atomic Overhaul > Krastorio 2 > Titanium > Zirconium". I'm wondering if there is supposed to be a different way to unlock the centrifuge or if there is some wonky mod interaction.
If you could help find the problem, I would greatly appreciate it.

Thank you very much,
Josiah

P.S.
This is my map string so you can see my mods (if that helps):

>>>eNp1UT2IE0EUnucas+aiXLGNIGeQa1fwtLDQ7CiIiGiXxsrJ7uRu
YH/C7CziXXEprlRstNDGa22usxAsAoIoKBxa2Z3YWHga8dBGiDO7mWS
yuXvw3n7zvTffe28WEKBLKLeep9ypkzCLWJxFbsIpQn1PZw77CQlLRB
jog/Kan3S7lJcvHvV5FlA3YVO3azSm0T23TVJZjCWFctpe5qS7wsSUg
rWSCLORzSLK01KRzXgSl3vbISVBmTwm96OchO5dIig3M3OccOpGVJAw
NflKKpJ4SqMq2EyzumCCxGz26SqCU5qaG9RFFi+ngs5ozGV8X4kjq4z
7SWxSCNxXe7u9jQWkfLiOGsOhcol25HMqR9CTlRiB5LRV/JB1Ogg1Lr
darStKCADWnK1rX1YfeVDUnMEjMBgx/bZmrmtwCx+YWtTgvKFzLrffB
iiaCtliVGXjCSiSGyoJ8GD3xebfN9tN+Pf818eb7TsenL3q/BgsbTVl
sqr2PDQOT58oe6lXQVpzxxulPnvw4b2y7x5U1A1HBXxBhv4NC8H8cYk
278vQOIn0aE0t42Do5PZHb/JVg09eeQ/5EBeV+IIKb1XIG44ngwLihx
jwKZ09MSmR95eQOUMw2fCdbvva6F8aZPZHmHuUmEW8z2+oqYbBOHyzx
tPI99yu6hN+hsFSQFXtSa44qYyWKr7zOH9uC01s4KHTj2///A+lmevp
<<<

**Comment by Xman1109** [comment]
I will take a look at it.

**Reply by josiahwilkinson** [reply]
Apologies, I've been away for a while. The issue appears to be with Brevven's Graphite & Diamonds mod also having a "Graphite Processing" tech that seems to override yours (I'm assuming it's due to the name). Due to this, your tech is dropped. I would assume (though I may be wrong), that one of you guys would need to change the name of the tech to avoid the overwriting.

Here is the save file (if the upload worked):
[Krastorio 2 plus Brevven.zip](https://github.com/Xman1109/Atomic_Overhaul/files/9215769/Krastorio.2.plus.Brevven.zip)

**Reply by Xman1109** [reply]
Yeah, that's possible, I will try my best to fix it, but it could need a couple of hours or days.

**Reply by Xman1109** [reply]
Thanks for bringing this to my attention.

**Reply by Xman1109** [reply]
Ok, I've updated the mod now. 
You can now research the graphite processing technology from AO after you've researched the one from Brevvens mod.
But you may need to research the last one again, because of Migration.
Btw, nice main Bus in your base ^^

**Reply by josiahwilkinson** [reply]
Thanks a lot man!
Just updated and loaded it up and it's all working, yours actually stayed researched and Brevven's got set to unreseached (probably because your mod was first in the priority listing).
Ha, thanks; it's just a mall supply while I store up for the main base. Brevven's mods and K2 add a lot of stuff.

**Reply by Xman1109** [reply]
I am glad I could help.
Thanks again for bringing this issue to my attention.
Have a great day!

**Comment by Xman1109** [comment]
The recipe of the centrifuge should be unlocked via the Graphite processing technology as seen in the picture below
![image](https://user-images.githubusercontent.com/72877576/179945424-92482a96-2cb6-4151-ac4e-f0b7ff71429f.png)

You should be able to craft it after researching it.
![image](https://user-images.githubusercontent.com/72877576/179946276-27cde19f-f6b3-4407-86ae-8248f5968a81.png)

**If this is not the case then please send me your `mod-list.json` and maybe the `mod-settings.dat`, so I can try to replicate the issue.** _(both are located in %appdata%/Factorio/mods)_
